src/SME.SGP.Dados/Repositorios/RepositorioPendenciaAulaConsulta.cs

```diff
@@ -193,9 +193,13 @@ public async Task<Turma> ObterTurmaPorPendencia(long pendenciaId)
 
         public async Task<IEnumerable<PendenciaAulaDto>> ObterPendenciasAulasPorPendencia(long pendenciaId)
         {
-            var query = @"select a.data_aula as DataAula, pa.Motivo
+            var query = @"select a.data_aula as DataAula, pa.Motivo, (a.tipo_aula = 2) ehReposicao, a.turma_id TurmaId, a.ue_id UeId, a.disciplina_id DisciplinaId, aa.nome_avaliacao as TituloAvaliacao
                            from pendencia_aula pa
-                          inner join aula a on a.id = pa.aula_id
+                           join pendencia p on p.id = pa.pendencia_id
+                           join aula a on a.id = pa.aula_id
+                           left join atividade_avaliativa aa on aa.turma_id = a.turma_id 
+   									and aa.data_avaliacao::date = a.data_aula::date    									
+									and a.professor_rf = aa.professor_rf and p.tipo = 10                           
```

Comment:
evitar uso de magic numbers
passe o valor por parametro

Reply:
Mas só se aplica a avaliação quando o tipo for 10 e não vejo motivo por passar essa informação por parâmetro, pois pode retornar avaliações para outros tipos.

Reply:
Entendi. Vou ajustar conforme o pedido da arquitetura.